extendr-api/src/functions.rs

```diff
@@ -385,8 +385,14 @@ pub fn na_str() -> &'static str {
 ///   Ok(())
 /// })
 /// ```
+#[cfg(feature = "extendr_engine")]
```

Comment:
Oh, I see, this code is the problem. The annotation here should be `#[cfg(test)]`, I think.